pkg/install/1-installresources.go

```diff
@@ -320,17 +320,17 @@ func (i *Installer) installResources(ctx context.Context) error {
 								InterfaceIPConfigurationPropertiesFormat: &mgmtnetwork.InterfaceIPConfigurationPropertiesFormat{
 									LoadBalancerBackendAddressPools: &[]mgmtnetwork.BackendAddressPool{
 										{
-											ID: to.StringPtr("[resourceId('Microsoft.Network/loadBalancers/backendAddressPools', '" + infraID + "-public-lb', '" + infraID + "-public-lb-control-plane')]"),
+											ID: to.StringPtr("[resourceId('Microsoft.Network/loadBalancers/backendAddressPools', '" + infraID + "-public-lb', '" + infraID + "-public-lb-control-plane-v4')]"),
 										},
 										{
-											ID: to.StringPtr("[resourceId('Microsoft.Network/loadBalancers/backendAddressPools', '" + infraID + "-internal-lb', '" + infraID + "-internal-controlplane')]"),
+											ID: to.StringPtr("[resourceId('Microsoft.Network/loadBalancers/backendAddressPools', '" + infraID + "-internal-lb', '" + infraID + "-internal-controlplane-v4')]"),
 										},
 									},
 									Subnet: &mgmtnetwork.Subnet{
 										ID: to.StringPtr(i.doc.OpenShiftCluster.Properties.MasterProfile.SubnetID),
 									},
 								},
-								Name: to.StringPtr("bootstrap-nic-ip"),
+								Name: to.StringPtr("bootstrap-nic-ip-v4"),
```

Comment:
This means all existing cluster will never get ipv6 :/ or will get it in a strange hybrid snowflaky form...

Reply:
I worked through these changes - all they do is keep us close to the installer, I don't think they limit our options (further).  If we do IPv6 at some point it is probably already likely that we would not try to retrofit it to existing clusters.